addons/portal/data/mail_template_data.xml

```diff
@@ -41,9 +41,6 @@
                             <a t-att-href="object.user_id.signup_url" style="display: inline-block; padding: 10px; text-decoration: none; font-size: 12px; background-color: #875A7B; color: #fff; border-radius: 5px;">
                                 <strong>Activate Account</strong>
                             </a>
-                            <a href="/web/login" style="display: inline-block; padding: 10px; text-decoration: none; font-size: 12px;">
-                                <strong>Log in</strong>
-                            </a>
```

Comment:
Is it really useful to remove this? It just makes the entire thing less clear and convenient, the user will not necessarily assume "Activate Account" will lead to the login screen.

Reply:
This is what was specified in the task. Will see what lna has to say during the review